docs/conf.py

```diff
@@ -190,6 +190,7 @@ def setup(app):
 github_map = {'dwavebinarycsp': 'dwavebinarycsp',
               'cloud': 'dwave-cloud-client',
               'dimod':  'dimod',
+              'gate': 'dwave-gate',
```

Comment:
I think the error is related to this line.

Reply:
Maybe I'm missing something obvious here, but I don't see why that would be the case? 

All the generated URLs seem to be valid. 🤔

Reply:
The failure is due to [this removal](https://github.com/dwavesystems/dwave-ocean-sdk/pull/247#discussion_r1088386301): the ``sphinx.ext.linkcode`` needs the version, which is calculated as ``pkg_resources.get_distribution('dwave-ocean-sdk').requires()``. 

You can get the build to pass by adding it manually as ``versions['dwave-gate'] = '0.1.0'``. Until you make ``dwave-gate`` a requirement of the SDK, you should set it to automatically add the version of ``dwave-gate``.